benchopt/runner.py

```diff
@@ -205,6 +205,78 @@ def get_next(x): return max(x / rho, MIN_TOL)
     return curve, status
 
 
+class Callback:
```

Comment:
This `Callback` should be private as it is not part of the private API.
```suggestion
class _Callback:
```